extensions/linz/schema.json

```diff
@@ -3,25 +3,76 @@
   "$id": "https://linz.github.io/stac/_STAC_VERSION_/linz/schema.json",
   "title": "LINZ STAC Extension",
   "description": "LINZ STAC Extension.",
-  "allOf": [
+  "oneOf": [
     {
-      "$ref": "https://schemas.stacspec.org/v1.0.0/collection-spec/json-schema/collection.json#"
-    },
-    {
-      "$ref": "https://linz.github.io/stac/_STAC_VERSION_/quality/schema.json#"
-    },
-    {
-      "$ref": "https://stac-extensions.github.io/version/v1.0.0/schema.json#"
-    },
-    {
-      "$ref": "https://stac-extensions.github.io/projection/v1.0.0/schema.json#"
+      "$comment": "This is the schema for STAC Items.",
+      "allOf": [
+        {
+          "$ref": "https://schemas.stacspec.org/v1.0.0/item-spec/json-schema/item.json#"
+        },
+        {
+          "$ref": "#/definitions/linz:item"
+        }
+      ]
     },
     {
-      "$ref": "#/definitions/linz"
+      "$comment": "This is the schema for STAC Collections.",
+      "allOf": [
+        {
+          "$ref": "https://schemas.stacspec.org/v1.0.0/collection-spec/json-schema/collection.json#"
+        },
+        {
+          "$ref": "https://linz.github.io/stac/_STAC_VERSION_/quality/schema.json#"
+        },
+        {
+          "$ref": "https://stac-extensions.github.io/version/v1.0.0/schema.json#"
+        },
+        {
+          "$ref": "https://stac-extensions.github.io/projection/v1.0.0/schema.json#"
+        },
+        {
+          "$ref": "#/definitions/linz:collection"
+        }
+      ]
     }
   ],
   "definitions": {
-    "linz": {
+    "linz:item": {
+      "type": "object",
+      "required": ["linz:geospatial_type"],
+      "properties": {
+        "linz:geospatial_type": {
+          "type": "string",
+          "enum": [
+            "Point",
+            "Linestring",
+            "Polygon",
+            "Multipoint",
+            "Multilinestring",
+            "Multipolygon",
+            "Geometry Collection",
+            "Circular String",
+            "Compound Curve",
+            "Curve Polygon",
+            "Multicurve",
+            "Multisurface",
+            "Polyhedral Surface",
+            "Triangle",
+            "Tin",
+            "Geometry",
+            "Grid",
+            "Greyscale",
+            "RGB",
+            "Multispectral",
+            "Hyperspectral",
+            "Colour Image",
+            "Black and White Image",
+            "Point Cloud"
```

Comment:
Is the order here relevant? Otherwise, would it be OK to order it alphabetically?

Reply:
Order isn't relevant. I think alphapbetical is fine.